programs/ziti-edge-tunnel/ziti-edge-tunnel.c

```diff
@@ -1852,8 +1855,6 @@ static void run(int argc, char *argv[]) {
     if (!scm_grant_se_debug()){
         ZITI_LOG(WARN, "could not set se debug access token on process. if process posture checks seem inconsistent this may be why");
     }
-#else
-    ziti_log_init(ziti_loop, ZITI_LOG_DEFAULT_LEVEL, NULL);
```

Comment:
probably not a big change/diff here... i just moved this up to the previous block where the log init happens to keep the related logic closer